atc/worker/container_provider.go

```diff
@@ -171,6 +171,7 @@ func (p *containerProvider) FindOrCreateContainer(
 
 			if !acquired {
 				time.Sleep(creatingContainerRetryDelay)
+				logger.Debug(fmt.Sprintf("did-not-acquire-creating-container-lock-%d", creatingContainer.ID()))
```

Comment:
this should use `lager.Data` instead of putting it in the message itself

Reply:
thanks. Made the change